src/main/java/ch/njol/skript/effects/EffCommandBlockConditional.java

```diff
@@ -0,0 +1,59 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.block.Block;
+import org.bukkit.block.data.type.CommandBlock;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Conditional / Unconditional")
+@Description(
+	"Sets whether the provided command blocks are conditional or not."
+)
+@Examples({
+	"make {_block} conditional",
+	"make {_block} unconditional if {_block} is conditional"
+})
+@Since("INSERT VERSION")
+public class EffCommandBlockConditional extends Effect {
+
+	static {
+		Skript.registerEffect(EffCommandBlockConditional.class, "make %blocks% [not:(un|not )]conditional");
```

Comment:
I'm _slightly_ worried about how these patterns have so little to do with command blocks, that on their own it's sort of hard to understand what they're doing, if that makes sense? It's kind of hard to guess what `make {var} conditional` is actually going to do.

If you can't think of anything then don't worry it's not a huge issue, I just feel like it could benefit from some context somehow.

Reply:
Yeah, I was thinking this when I made it, but I couldn't think of another way to word it without sounding clunky. Would love to hear any ideas if anyone has one.

Reply:
Thoughts on `make command block[s] %blocks%...`?

Reply:
i like it